static/js/click_handlers.js

```diff
@@ -673,13 +671,15 @@ export function initialize() {
         popovers.hide_all();
     }
 
-    $("#compose_buttons").on("click", handle_compose_click);
-    $(".compose-content").on("click", handle_compose_click);
+    $("body").on("click", "#compose_buttons", handle_compose_click);
+    $("body").on("click", ".compose-content", handle_compose_click);
 
-    $("#compose_close").on("click", () => {
+    $("body").on("click", "#compose_close", () => {
         compose_actions.cancel();
     });
```

Comment:
These buttons are kinda the closed-compose widget, which in theory we could leave in the backend templates.

Reply:
I don't understand what you mean here. What do you mean by `closed-compose widget`? By backend templates, do you mean `html` files?